lib/opal/nodes/call.rb

```diff
@@ -8,7 +8,7 @@
 module Opal
   module Nodes
     class CallNode < Base
-      handle :send, :csend
```

Comment:
here I see a revert of the csend patch, should definitely go into a separate commit imo

Reply:
unless we could find an edge case that causes the issue...

Reply:
As we now know, that its not the new CDP runner code that causes the issues, but instead #2467 , i simply removed this